pcsx2/USB/usb-turntable/usb-turntable.cpp

```diff
@@ -0,0 +1,488 @@
+/*  PCSX2 - PS2 Emulator for PCs
+ *  Copyright (C) 2002-2023 PCSX2 Dev Team
+ *
+ *  PCSX2 is free software: you can redistribute it and/or modify it under the terms
+ *  of the GNU Lesser General Public License as published by the Free Software Found-
+ *  ation, either version 3 of the License, or (at your option) any later version.
+ *
+ *  PCSX2 is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ *  without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+ *  PURPOSE.  See the GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License along with PCSX2.
+ *  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "PrecompiledHeader.h"
+
+#include "Host.h"
+#include "Input/InputManager.h"
+#include "StateWrapper.h"
+#include "USB/USB.h"
+#include "USB/deviceproxy.h"
+#include "USB/qemu-usb/USBinternal.h"
+#include "USB/qemu-usb/desc.h"
+#include "USB/usb-turntable/usb-turntable.h"
+#include "USB/usb-pad/usb-pad.h"
+
+namespace usb_turntable
+{
+	enum
+	{
+		STR_MANUFACTURER = 1,
+		STR_PRODUCT_MOUSE,
+		STR_PRODUCT_TABLET,
+		STR_PRODUCT_KEYBOARD,
+		STR_SERIALNUMBER,
+		STR_CONFIG_MOUSE,
+		STR_CONFIG_TABLET,
+		STR_CONFIG_KEYBOARD,
+	};
```

Comment:
I can't see this enum referenced anywhere, could it be removed?

Reply:
Oops yeah, that was copied when i cloned one of the other hid devices, removed now